core/db/seeds.rb

```diff
@@ -1,5 +1,8 @@
 # Loads seed data out of default dir
 default_path = File.join(File.dirname(__FILE__), 'default')
 
-Rake::Task['db:load_dir'].reenable
-Rake::Task['db:load_dir'].invoke(default_path)
+['stores', 'store_credit', 'countries', 'return_reasons', 'states',
+ 'stock_locations', 'zones', 'refund_reasons', 'roles', 'shipping_categories'].each do |seed|
```

Comment:
Yes. I also would like to have this separated by line 👍 

And maybe use `%w()` syntax, but I don't care about this very much.